Source/AppScaffolding/AppScaffolding.csproj

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <TargetFramework>net6.0-windows</TargetFramework>
-    <Version>7.7.1.1</Version>
```

Comment:
Version shouldn't get smaller

Reply:
Yeah, I'll revert this change. This is a leftover from when I was testing the automatic upgrader.